lib/auth/auth_with_roles_test.go

```diff
@@ -6679,8 +6680,12 @@ func TestWatchHeadlessAuthentications_usersCanOnlyWatchThemselves(t *testing.T)
 		},
 	}
 
+	// Initialize headless watcher for each test cases.
 	for _, tc := range testCases {
-		t.Run(tc.name, func(t *testing.T) {
+		tc := tc
+		t.Run(fmt.Sprintf("%v/init_watcher", tc.name), func(t *testing.T) {
```

Comment:
The go tool automatically adds a / character for subtests, so by manually adding one here in the name it will end up looking like an extra level of subtest.